web/src/app/shared/models/profile.model.ts

```diff
@@ -1,4 +1,4 @@
-import { ActivityModel } from './activity.model';
+import { ActivityModel } from './index.model';
```

Comment:
The models themselves import the files directly. This change is not required

Reply:
done